docs/pages/includes/config-reference/auth-service.yaml

```diff
@@ -383,6 +383,13 @@ auth_service:
     # in a leaf cluster. Defaults to false.
     load_all_cas: false
 
+    # Enables case-insensitivity for ssh dial requests. Setting this to true will allow
+    # 'tsh ssh myserver' to route to host 'MyServer'. Dialing with an uppercase hostname
+    # will still require an exact match. Openssh always lowercases hostnames, so setting
+    # this to true is necessary if you want to use openssh to access nodes with uppercase
+    # characters in their hostnames.
+    case_insensitive_routing: false
```

Comment:
cc @ptgott is there any other OpenSSH-related documentation we might want to update?

Reply:
We mention the `tsh ssh` command throughout the docs, but the only page I can think of that focuses on OpenSSH is `docs/pages/server-access/guides/openssh.mdx`.

I think it would be good to mention case insensitive routing in `docs/pages/connect-your-client/tsh.mdx`.

CC @fspmarshall